front/README.md

```diff
@@ -57,14 +57,19 @@ You may pass the same options and arguments to this script as you would to `yarn
 > - if on Linux, you can also launch all the containers on the host network: you can replace the
 >   `docker compose <something>` above with `osrd/scripts/host-compose.sh <something>`
 
-If the tests fail, you'll find a `front/test-results` folder that will contain videos of the fail
-test executions. They might be of help to understand what's going on. Note that the CI also exports
-these videos as artifacts.
+If the tests fail, a `front/test-results` folder will be created, containing videos and traces of the failed test executions.
+These files can help you understand what went wrong.
+Additionally, the CI system exports these videos and traces as artifacts.
+You can view the trace files using the [Playwright trace viewer](https://trace.playwright.dev/).
```

Comment:
Nit: line wrapping is a bit off. In general we don't wrap the README by sentence, we wrap at 80 characters.